internal/config/cgmgr/systemd.go

```diff
@@ -70,8 +72,16 @@ func (*SystemdManager) MoveConmonToCgroup(cid, cgroupParent, conmonCgroup string
 		cgroupParent = conmonCgroup
 	}
 	conmonUnitName := fmt.Sprintf("crio-conmon-%s.scope", cid)
+
+	// Set the systemd KillSignal to SIGPIPE that conmon ignores.
+	// This helps during node shutdown so that conmon waits for the container
+	// to exit and doesn't forward the SIGTERM that it gets.
+	killSignalProp := systemdDbus.Property{
+		Name:  "KillSignal",
+		Value: dbus.MakeVariant(13),
```

Comment:
Maybe
 - use `int(unix.SIGPIPE)` as otherwise it's a non-descriptive magic number
 - add a comment explaining that SIGPIPE is ignored, and it is set here so conmon can ignore termination signal from systemd.

Reply:
(second issue fixed while I was writing my review comment, so please ignore)

Reply:
Switched to using `int(unix.SIGPIPE)`. Thanks!